src/en/components/footer/code.md

```diff
@@ -39,15 +42,22 @@ Choose the **full display** if you need to include:
 - Add content to the contextual band by using both the `contextual-heading` and `contextual-links` attributes.
 - Use the `contextual-heading` attribute to assign heading text and the navigational landmark label of the contextual band.
 - Use the `contextual-links` attribute to set a maximum of three links in the contextual band. Add support links for your product or service, such as contact details for a program.
-- Set the `contextual-links`, by passing an object or an object in a string in the following format.
+- For the contextual band, set the `contextual-links`, by passing an object or an object in a string in the following format:
+```js
+{
+“1st link label”: “href”
+“2nd link label”: “href”
+“3rd link label”: “href”
+}
+```
+- For the main band, set the `sub-links`, by passing an object or an object in a string in the following format:
```

Comment:
"For the main band" should be "For the footer links band"

Reply:
Updated